projects/fsastorefrontlib/src/cms-components/my-account/premium-calendar/premium-calendar.component.html

```diff
@@ -1,238 +1,134 @@
+            (click)="toggleActiveAccordion(i)"
+            [ngClass]="{
+              clickable: policy.billingData?.length > 1,
+              active: selectedIndexes.includes(i)
+            }"
+          >
+            <div class="col">
+              <div class="row">
+                <div class="col-4">
+                  <div class="image-wrapper small">
+                    <img
+                      class="image-item"
+                      [src]="
+                        getBaseUrl() +
+                        policy.mainProduct.coverageProduct.images[0]?.url
+                      "
+                      alt=""
+                    />
+                  </div>
+                </div>
+                <div class="col-8 mw-100">
+                  {{ policy.categoryData?.name }} <br />
+                  {{ policy.contractNumber }}
                 </div>
               </div>
-              <div class="col-8">
-                {{ policy.categoryData?.name }} <br />
-                {{ policy.contractNumber }}
+            </div>
+            <div class="col">
+              <h5 class="d-lg-none m-0">
+                {{ 'fscommon.paymentFrequency' | cxTranslate }}
+              </h5>
+              {{ policy.paymentFrequency }}
+            </div>
+            <div
+              class="col text-left"
+              [ngClass]="
+                policy.billingData[0]?.paymentStatus == 'Overdue'
+                  ? 'warn'
+                  : 'notice'
+              "
+            >
+              <h5 class="d-lg-none p-0 m-0">
+                {{ 'premiumCalendar.dueDate' | cxTranslate }}
+              </h5>
+              {{ policy.billingData[0]?.dueDate | cxDate: 'dd MMM yyyy' }}
+              <div
+                class="warn"
+                *ngIf="policy.billingData[0]?.paymentStatus == 'Overdue'"
+              >
+                {{ 'premiumCalendar.overdue' | cxTranslate }}
               </div>
             </div>
-          </div>
-          <div class="col">
-            {{ policy.paymentFrequency }}
-          </div>
-          <div
-            class="col"
-            [ngClass]="
-              policy.billingData[0]?.paymentStatus == 'Overdue'
-                ? 'warn'
-                : 'notice'
-            "
-          >
-            {{ policy.billingData[0]?.dueDate | cxDate: 'dd MMM yyyy' }}
             <div
-              class="warn"
-              *ngIf="policy.billingData[0]?.paymentStatus == 'Overdue'"
+              class="col"
+              [ngClass]="
+                policy.billingData[0]?.paymentStatus == 'Overdue'
+                  ? 'warn'
+                  : 'notice'
+              "
             >
-              {{ 'premiumCalendar.overdue' | cxTranslate }}
+              <h5 class="d-lg-none p-0 m-0">
+                {{ 'fscommon.premium' | cxTranslate }}
+                <span
+                  class="position-absolute fs-icon icon-triangle-bottom"
```

Comment:
This span should be outside of this H5. It should be on the same level as H5. If not the arrow may not be centered right. Also you can remove `position-absolute`, and in CSS for this element include our vertical-center mixin.